--- vlc-android/compile-libvlc.sh	2017-10-24 21:58:23.000000000 +0200
+++ vlc-android/compile-libvlc-new.sh	2017-10-24 22:38:54.000000000 +0200
@@ -76,7 +76,7 @@
     --disable-sdl \
     --disable-SDL_image \
     --disable-fontconfig \
-    --enable-zvbi \
+    --disable-zvbi \
     --disable-kate \
     --disable-caca \
     --disable-gettext \
@@ -144,7 +144,7 @@
     --enable-mpg123 \
     --disable-dca \
     --disable-sdl-image \
-    --enable-zvbi \
+    --disable-zvbi \
     --disable-fluidsynth \
     --enable-fluidlite \
     --disable-jack \
